### PR TITLE
Disable sliders

### LINKED ISSUE #424: downloading a jupyter notebook has html does not have the image/plot included
@davis278 found the issue.

To reproduce the problem:
conda create -n just_vcs -c cdat/label/nightly -c conda-forge vcs jupyter "libnetcdf=4.6.2" "python=3.7"
source activate just_vcs
jupyter notebook

open a notebook for the kernel for this env, and run the following in the notebook cells:
```
import vcs
import cdms2
vcs.download_sample_data_files()
f=cdms2.open(vcs.sample_data+"/clt.nc")
s=f("clt")
x=vcs.init(bg=True)
x.plot(s)
```

you should see a plot.
click on 'File' tab, and hover over to 'Download as', and select 'HTML', it should download the notebook as html, and viewing the html in a browser should show the plot, but it does not.

### PR DESCRIPTION
**Pull request by forsyth2**
Disabling sliders until we can implement them properly. This resolves #424 and replaces #426. Tests are in https://github.com/CDAT/jupyter-vcdat/pull/154

### CODE REVIEW

File: test_export_html.ipynb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.3"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```
Comment (forsyth2) [inline test_export_html.ipynb]: Add newline

File: tests/test_vcs_export_html.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import os
```
Comment (forsyth2) [inline tests/test_vcs_export_html.py]: Is this file the best way to test that the html displays the plots properly?

File: vcs/displayplot.py [hunk 3]
```diff
@@ -341,18 +359,7 @@ def _repr_png_(self):
                     IPython.display.display(vbox)
             else:
                 IPython.display.clear_output()
-            tmp = tempfile.mktemp() + ".png"
-            self._parent.png(tmp)
-            f = open(tmp, "rb")
-            st = f.read()
-            f.close()
-            self._parent._display_target_image.value = st
```
Comment (forsyth2) [inline vcs/displayplot.py]: I don't think we need this line outside of the `SLIDERS_ENABLED` section above.

**Comment by forsyth2** [comment]
Will squash commits before merging.

File: tests/test_vcs_export_html.py [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+                image_tag_present = True
+    assert image_tag_present
+
+
+class TestVCSExportHTML(unittest.TestCase):
+    def testExportHTML(self):
+        debug_print('Run the notebook')
+        # This is apparently NOT equivalent to running the notebook on the web.
+        # Generating the HTML this way keeps the plot, whereas running from the web does not.
+        assert os.system('jupyter nbconvert --to notebook --execute ../test_export_html.ipynb') == 0
```
Comment (forsyth2) [inline tests/test_vcs_export_html.py]: Circle CI apparently expects a different file path.

**Comment by muryanto1** [comment]
@forsyth2 please pull the code in master to your branch, your branch's .circleci/config.yml is outdated which is why it is failing in circleci.

File: .circleci/config.yml [hunk 5]
```diff
@@ -57,9 +57,9 @@ aliases:
   - &conda_upload
     name: conda_upload
     command: |
-       if [[ $CIRCLE_BRANCH != "master" ]]; then
-          exit 0
-       fi
+       #if [[ $CIRCLE_BRANCH != "master" ]]; then
```
Comment (forsyth2) [inline .circleci/config.yml]: Revert changes in this file before merging.

**Comment by forsyth2** [comment]
@downiec @muryanto1 Ready for review. I will revert the second commit before merging.

File: .circleci/config.yml (hunk 5 above)
Comment (downiec) [reply .circleci/config.yml]: Looks like you need to revert these changes?

**Pull request merged**